src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLSPTextDocumentCreatedListener.cs

```diff
@@ -14,17 +17,23 @@ namespace Microsoft.VisualStudio.LanguageServerClient.Razor
     [Export(typeof(RazorLSPTextDocumentCreatedListener))]
     internal class RazorLSPTextDocumentCreatedListener
     {
+        private static readonly Guid HtmlLanguageServiceGuid = new Guid("9BBFD173-9770-47DC-B191-651B7FF493CD");
```

Comment:
NIT; maybe a comment for where this GUID was sourced from.

Reply:
Can't reference the source because it's not open source. Plus the name serves as a good enough description. We do this in [other places](https://github.com/dotnet/aspnetcore-tooling/blob/master/src/Razor/src/Microsoft.VisualStudio.RazorExtension/RazorPackage.cs#L28) as well.